.ci/xenial.jenkinsfile

```diff
@@ -166,9 +186,26 @@ node('Linux') {
         echo 'Run tests that check binaries work in compatible images'
     }
 
-    // TODO: If master branch
-    stage('Upload images') {
-        echo 'Upload images'
-    }
+
+    //if (isMaster) {
+        // If it was master and everything is ok up to here
+        stage('Upload images') {
+            List images = ["base-${distro}"]
+            gccVersions.each {
+                images.add("gcc${versionMajor}-${distro}")
+                images.add("gcc${versionMajor}-${distro}-jenkins")
+            }
+            clangVersions.each {
+                images.add("clang${versionMajor}-${distro}")
+                images.add("clang${versionMajor}-${distro}-jenkins")
+            }
+
+            images.each { image ->
+                String uploadImage = "${originalUsername}/${image}:${conanVersion}"
+                sh "docker tag ${dockerUsername}/${image}:${branchName}-${commit} ${uploadImage}"
+                // sh "docker push ${uploadImage}"
```

Comment:
I want to check this works (image names are properly set) before activating the `push` (see also that the conditional `isMaster` is commented out).